src/main/java/service/imp/WriterServiceImpl.java

```diff
@@ -0,0 +1,17 @@
+package service.imp;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import service.WriterService;
+
+public class WriterServiceImpl implements WriterService {
+    @Override
+    public void write(String resultingReport, String fileName) {
```

Comment:
Avoid hardcoding the file path in the service layer. It is better to receive the file path as a parameter, which you are already doing, but make sure not to hardcode it anywhere else.